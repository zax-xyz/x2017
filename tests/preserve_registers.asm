FUNC LABEL 0
    MOV REG 0 VAL 10
    MOV REG 1 VAL 50
    MOV REG 2 VAL 100
    CAL VAL 1
    PRINT REG 0
    PRINT REG 1
    PRINT REG 2
    RET
FUNC LABEL 1
    PRINT REG 0
    PRINT REG 1
    PRINT REG 2
    MOV REG 0 VAL 5
    MOV REG 1 VAL 2
    MOV REG 2 VAL 200
    RET
