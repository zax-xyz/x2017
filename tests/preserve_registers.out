10
50
100
5
2
200
